src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +100,8 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+
+  public int amountOfConversations(){
```

Comment:
Lets add a method descriptor here as well.

Reply:
Also naming suggestion "getConversationCount". Lol, on the next review I'll make sure not to point on things like this, its not important and it really just my preference.